sonar-text-dotnet/src/SonarLint.Secrets.DotNet.UnitTests/DotNetOnly/Readme.txt

```diff
@@ -0,0 +1,7 @@
+﻿DotNetOnly test code
+
+These tests are either:
+
+1) tests for a .NET class that doesn't have an equivalent Java class, or 
+
+2) tests for a ported Java class for which there are no Java classes.
```

Comment:
FYI I think I meant "tests for a ported Java class for which there are no Java ~~classes~~ **tests**."